### ISSUE TITLE
UAC prompt blocked when Cairo is active

### ISSUE THREAD

**Issue by schade** [cairoshell/cairoshell#61]
When installing any application under (at least) recent builds of Windows 10, the UAC prompt does not come to the foreground unless you exit Cairo and return to the Windows shell. At this point you can see the prompt as a taskbar item and select it to bring the prompt to the foreground. You can immediately relaunch Cairo after accepting the UAC warning with no ill effects.

**Comment by dremin** [comment]
Hey @schade - I haven't seen any problems with UAC prompts coming to the foreground on my Windows 10 machines. A couple questions to help me reproduce-- what build of Windows 10 are you using? Is there a certain installer that causes this, and how are you launching that installer?

As a workaround, are you able to access the UAC prompts using alt-tab?

**Comment by schade** [comment]
I'm on the slow ring of the Insider program running build 17025 currently. When I noticed the issue a few days ago it was on Cairo Desktop M3.6540; I noticed the update available to M3.6545 a few minutes ago and after updating the issue appears to be fixed now. 

For reference if this pops back up I noticed this issue on both the 2.7 and 3.6 Python installers, and the Armored Warfare (not Steam) installer. As the issue appears to be fixed, this issue should be closed.
